tests/Operation/ChangeStreamFunctionalTest.php

```diff
@@ -129,4 +129,29 @@ public function test_resume_after_kill_then_insert()
         $changeStreamResult->next();
         $this->assertNull($changeStreamResult->current());
     }
+
+    public function test_key()
+    {
+        $operation = new DatabaseCommand("admin", ["setFeatureCompatibilityVersion" => "3.6"]);
```

Comment:
I think this came up in a previous review, but we should expect the server to have the correct FCV already. The `setUp()` method's skip logic should handle cases where the server is misconfigured.